actionmailbox/app/controllers/rails/conductor/action_mailbox/inbound_emails_controller.rb

```diff
@@ -20,14 +20,18 @@ def create
 
     private
       def new_mail
-        Mail.new(params.require(:mail).permit(:from, :to, :cc, :bcc, :x_original_to, :in_reply_to, :subject, :body).to_h).tap do |mail|
+        Mail.new(mail_params.to_h.except(:attachments)).tap do |mail|
```

Comment:
```suggestion
        Mail.new(mail_params.except(:attachments).to_h).tap do |mail|
```

We're calling `to_h` because `Mail.new` requires it; doing the conversion last makes that clearer, IMO.

Reply:
This has been done and confirmed still working. Thanks for the suggestion!